### DISCUSSION TITLE
[Resolved] Plan on updating?

### DISCUSSION THREAD

**Discussion by mandyntx** [Drumstix42/drumstix42.github.io#8] [General]
Hello! The Icarus resource calculator is great but a bit outdated. Any plans to bring it up to date? :(

https://drumstix42.github.io/icarus/

**Comment by Drumstix42** [comment]
@mandyntx I know it's been a while, but I've updated to the latest Update (168).

Please take a look. I'll try to update it more regularly as I can.

**Reply by mandyntx** [reply]
Thank you!!
